### PR TITLE
Refactor the <Input> component

### PR DESCRIPTION
**Pull request by yuki24**
* [x] Add a test for the `inputRef` prop

### CODE REVIEW

File: src/lib/Components/Bidding/Components/Input.tsx [hunk 1]
```diff
@@ -51,12 +50,12 @@ export class Input extends Component<InputProps, InputState> {
...
         borderColor={this.state.borderColor}
         fontSize={3}
-        onBlur={() => this.onBlur()}
-        onFocus={() => this.onFocus()}
-        ref={component => this.props.createCustomRef && this.props.createCustomRef(this.props.refName, component)}
         p={3}
         pb={2}
         {...this.props}
+        ref={this.props.inputRef}
+        onBlur={() => this.onBlur()}
```
Comment (sweir27) [inline src/lib/Components/Bidding/Components/Input.tsx]: Ah, just sticking it below the `{...this.props}` is enough to make it override if an `onBlur` is passed in as props? Didn't realize that!

**Comment by sweir27** [comment]
@yuki24 from a quick glance this looks ✨ and way simpler than the implementation I tried! Maybe I can pick your brain about how you got the refs to work in this way? Although it seems to have just _worked_. 😄

**Comment by yuki24** [comment]
This PR is ready for review (except I was too lazy and didn't add details)

**Comment by sweir27** [comment]
Haha, we went over this in person! Looks 👍 and glad it could be simplified.

**Pull request merged**